File: Paladins.Api/Paladins.Api/Paladins.Common/Models/MatchDetailsModel.cs
using System.Collections.Generic;

namespace Paladins.Common.Models
{
    public class MatchDetailsModel
    {
        public int PaladinsMatchId { get; set; }
        public int PaladinsQueueId { get; set; }
        public int PaladinsPlayerId { get; set; }
        public string MapName { get; set; }
        public string PlayerName { get; set; }
        public int PlayerAccountLevel { get; set; }
        public string MapGameType { get; set; }
        public string HasReplay { get; set; }
        public int Length { get; set; }
        public int LengthInMinutes { get; set; }
        public string WinStatus { get; set; }
        public string Region { get; set; }
        public int TeamOneScore { get; set; }
        public int TeamTwoScore { get; set; }
        public int PaladinsChampionId { get; set; }
        public int PaladinsChampionSkinId { get; set; }
        public int SelfHealing { get; set; }
        public int GoldEarnedPerMinute { get; set; }
        public int GoldEarnedTotal { get; set; }
        public int HealingDone { get; set; }
        public int DamageDoneInHand { get; set; }
        public int DamageTaken { get; set; }
        public int DamageMitigated { get; set; }
        public int Deaths { get; set; }
        public int Kills { get; set; }
        public int KillingSpree { get; set; }
        public int KillsDouble { get; set; }
        public int KillsTriple { get; set; }
        public int KillsQuadra { get; set; }
        public int KillsPenta { get; set; }
        public int MultiKillMax { get; set; }
        public int LeagueLosses { get; set; }
        public int LeageWins { get; set; }
        public int LeagueTier { get; set; }
        public int LeaguePoints { get; set; }
        public IEnumerable<MatchBansModel> MatchBans { get; set; }
        public IEnumerable<ItemsBoughtModel> ItemsBought { get; set; }
        public IEnumerable<LoadoutSelectedModel> LoadoutSelected { get; set; }
    }
}

File: Mobile/app/src/main/java/com/fivegearszerochill/paladins/domain/models/match/MatchDetailsModel.kt
package com.fivegearszerochill.paladins.domain.models.match

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class MatchDetailsModel (
    @Expose
    @SerializedName("id")
    val id: Long,
    @Expose
    @SerializedName("paladinsMatchId")
    val paladinsMatchID: Long,
    @Expose
    @SerializedName("paladinsQueueId")
    val paladinsQueueID: Long,
    @Expose
    @SerializedName("paladinsPlayerId")
    val paladinsPlayerID: Long,
    @Expose
    @SerializedName("mapName")
    val mapName: String,
    @Expose
    @SerializedName("playerName")
    val playerName: String,
    @Expose
    @SerializedName("playerAccountLevel")
    val playerAccountLevel: Long,
    @Expose
    @SerializedName("mapGameType")
    val mapGameType: String,
    @Expose
    @SerializedName("hasReplay")
    val hasReplay: String,
    @Expose
    @SerializedName("length")
    val length: Long,
    @Expose
    @SerializedName("lengthInMinutes")
    val lengthInMinutes: Long,
    @Expose
    @SerializedName("winStatus")
    val winStatus: String,
    @Expose
    @SerializedName("region")
    val region: String,
    @Expose
    @SerializedName("teamOneScore")
    val teamOneScore: Long,
    @Expose
    @SerializedName("teamTwoScore")
    val teamTwoScore: Long,
    @Expose
    @SerializedName("paladinsChampionID")
    val paladinsChampionID: Long,
    @Expose
    @SerializedName("championName")
    val championName: String,
    @Expose
    @SerializedName("championIconUrl")
    val championIconUrl: String,
    @Expose
    @SerializedName("paladinsChampionSkinID")
    val paladinsChampionSkinID: Long,
    @Expose
    @SerializedName("selfHealing")
    val selfHealing: Long,
    @Expose
    @SerializedName("goldEarnedPerMinute")
    val goldEarnedPerMinute: Long,
    @Expose
    @SerializedName("goldEarnedTotal")
    val goldEarnedTotal: Long,
    @Expose
    @SerializedName("healingDone")
    val healingDone: Long,
    @Expose
    @SerializedName("damageDoneInHand")
    val damageDoneInHand: Long,
    @Expose
    @SerializedName("damageTaken")
    val damageTaken: Long,
    @Expose
    @SerializedName("damageMitigated")
    val damageMitigated: Long,
    @Expose
    @SerializedName("assists")
    val assists: Long,
    @Expose
    @SerializedName("objectiveAssists")
    val objectiveAssists: Long,
    @Expose
    @SerializedName("deaths")
    val deaths: Long,
    @Expose
    @SerializedName("kills")
    val kills: Long,
    @Expose
    @SerializedName("killingSpree")
    val killingSpree: Long,
    @Expose
    @SerializedName("killsDouble")
    val killsDouble: Long,
    @Expose
    @SerializedName("killsTriple")
    val killsTriple: Long,
    @Expose
    @SerializedName("killsQuadra")
    val killsQuadra: Long,
    @Expose
    @SerializedName("killsPenta")
    val killsPenta: Long,
    @Expose
    @SerializedName("multiKillMax")
    val multiKillMax: Long,
    @Expose
    @SerializedName("leagueLosses")
    val leagueLosses: Long,
    @Expose
    @SerializedName("leagueWins")
    val leagueWINS: Long,
    @Expose
    @SerializedName("leagueTier")
    val leagueTier: Long,
    @Expose
    @SerializedName("leaguePoints")
    val leaguePoints: Long,
    @Expose
    @SerializedName("matchBans")
    val matchBans: List<MatchBanModel>,
    @Expose
    @SerializedName("itemsBought")
    val itemsBought: List<ItemsBoughtModel>,
    @Expose
    @SerializedName("loadoutSelected")
    val loadoutSelected: List<LoadoutSelectedModel>
)

